### ISSUE TITLE
Call to undefined method xp::stringOf())

### ISSUE THREAD

**Issue by thekid** [xp-forge/web#57]
```bash
$ xp web -r static -
Uncaught exception: Error (Call to undefined method xp::stringOf())
  at <source> [line 395 of ...\Socket.class.php]
  at <main>('-r', 'static', '-') [line 0 of xp.web.Runner]
  at peer.Socket->toString() [line 34 of Standalone.class.php]
  at xp.web.srv.Standalone->serve('-', 'dev', io.Path{}, io.Path{}, array[0], array[0], '-') [line 123 of Runner.class.php]
  at xp.web.Runner::main(array[3]) [line 381 of class-main.php]
```

**Comment by thekid** [comment]
Fixed in xp-framework/networking@c45d32eb5413a79e7d7c5311dc6f915f52234a6c

**Comment by thekid** [comment]
Released in https://github.com/xp-framework/networking/releases/tag/v9.3.1
